app/src/main/kotlin/com/intive/picover/parties/view/AddPartyContent.kt

```diff
@@ -41,7 +42,16 @@ fun AddPartyBottomSheet(
 			onTitleChange = { viewModel.updateTitle(it) },
 			description = state.description,
 			onDescriptionChange = { viewModel.updateDescription(it) },
-			onSaveButtonClick = { navController.popBackStack() },
+			onSaveButtonClick = {
+				viewModel.addParty(
```

Comment:
shouldn't this be an event ?

Reply:
@marcinWisnia I am thinking of sending a PR for Repo now, and taking care of ViewModel within next one. Would that be ok?

Reply:
I am fully ok with splitting this into:
- repo
- VM 
parts :)

Reply:
Ok, in that case this and the other PR for 119 are going to be merged to [119/save-party-for-merge-with-dev](https://github.com/Android-Guild/Picover/tree/119/save-party-for-merge-with-dev)